lib/link_check.rb

```diff
@@ -3,11 +3,10 @@ class LinkCheck
 
   def initialize(uri)
     @uri = uri
-    @report = UriChecker::Report.new
+    @report = LinkCheck::UriChecker::Report.new
```

Comment:
these should still work without the namespace you've added in as it'll be scoped to `LinkCheck` already